build.sbt

```diff
@@ -7,15 +7,16 @@ name := """codacy-codesniffer"""
 
 version := "1.0-SNAPSHOT"
 
-val languageVersion = "2.12.7"
+val languageVersion = "2.13.1"
 
 scalaVersion := languageVersion
 
-resolvers ++= Seq("Typesafe Repo".at("http://repo.typesafe.com/typesafe/releases/"),
+resolvers ++= Seq("Typesafe Repo".at("https://repo.typesafe.com/typesafe/releases/"),
```

Comment:
You can remove resolvers here and use the default ones

Reply:
done